### PR TITLE
Update microsoft-secure-score-improvement-actions.md

### LINKED ISSUE #12554: Typo on the page
[Enter feedback here]
Under "Completed recommended actions" the first letter of the next paragraph should be capitalized.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 75a2c77d-f8c2-b663-57a1-1b1ec8565e89
* Version Independent ID: 69cc2b0f-d0d0-d4b1-8cb9-13c246d67e70
* Content: [Assess your security posture through Microsoft Secure Score](https://learn.microsoft.com/en-us/microsoft-365/security/defender/microsoft-secure-score-improvement-actions?view=o365-worldwide#take-action-to-improve-your-score)
* Content Source: [microsoft-365/security/defender/microsoft-secure-score-improvement-actions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/microsoft-secure-score-improvement-actions.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @siosulli
* Microsoft Alias: **siosulli**

### PR DESCRIPTION
**Pull request by msbemba**
Fixed Typo

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12554

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/microsoft-secure-score-improvement-actions.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ You won't be able to choose a status for Secure Score recommended actions in the
 
 #### Completed recommended actions
 
-recommended actions have a "completed" status once all possible points for the recommended action have been achieved. Completed recommended actions are confirmed though Microsoft data, and you can't change the status.
+Recommended actions have a "completed" status once all possible points for the recommended action have been achieved. Completed recommended actions are confirmed though Microsoft data, and you can't change the status.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/microsoft-secure-score-improvement-actions.md]:
```suggestion
Recommended actions have a "completed" status once all possible points for the recommended action have been achieved. Completed recommended actions are confirmed through Microsoft data, and you can't change the status.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**